account_asset_management/models/account_asset.py

```diff
@@ -519,12 +504,73 @@ def open_entries(self):
             "domain": [("id", "in", am_ids)],
         }
 
-    @api.multi
-    def compute_depreciation_board(self):
+    def _group_lines(self, table):
```

Comment:
Please add a docstring for this method. Its code is not obvious.

Reply:
A possible docstring is the one that is as a comment:

```
Group lines prior to depreciation start period.
```